### PR TITLE
South Africa-Alexander Msumba

### PR DESCRIPTION
**Pull request by MSUMBA-art**
# Your Details

- Your Name:
- Your City:
- Your Slack Name:

# Homework Details

- Module:
- Week:

### CODE REVIEW

**Review by theTrainMan — COMMENTED** [review]
Q 11
+ Just a note, that a pull request is to fork someone else's source code and making recommendations to alter or better the code?
Don't you think?

File: homework.md [hunk 1]
```diff
@@ -4,110 +4,110 @@
...
 
-You send the changes of your local repository to Github
+Push means to submit commits from your repository to GitHub 
 
 ## Question 14
 
 What is Github?
 
-A place for us to store and share our code.
+A web service where you can sign up and get and account
```
Comment (theTrainMan) [inline homework.md:90]: + I think you need to google this and try and be more specific..

File: homework.md [hunk 2]
```diff
@@ -4,110 +4,110 @@
...
 ## Question 16
 
 What program do we use to write code at CodeYourFuture?
 
-Microsoft Word
+Visual Studio Code
 
 ## Question 17
 
 What do you use "Commit Messages" for?
```
Comment (theTrainMan) [inline homework.md:106]: + The commit command is used to save changes to a local repository after staging in Git

File: homework.md [hunk 3]
```diff
@@ -4,110 +4,110 @@
...
 
 }
-```
 
 ## Question 7
 
 What is JavaScript?
 
-A programming language used on the internet
+A programming language commonly used in web development
```
Comment (Ntshembo-Hlongwane1) [inline homework.md:45]: When you say that JavaScript is a programming language commonly used in web development you are **right** and at the same time **wrong**.

So why am I seeing this you may wonder? So you have to understand that JavaScript over the past few years has evolved so much ever since **2009** when **NodeJS** was introduced we could now build **CLI's and Web Services** using JavaScript and come **2015 when React Native** was introduced we could build Native Mobile applications and finally **2019 ElectronJS** was introduced and we could build Desktop application for all OS's.

So basically JavaScript is not only for Web Dev

File: homework.md [hunk 4]
```diff
@@ -4,110 +4,110 @@
...
+Help to share information
+Enable people to make their own changes
+Allow us to make checkpoints so that we can save our work as we go along
 
 ## Question 9
 
 What happens when you `fork` a repository?
 
-You delete it
+You copy it
```
Comment (Ntshembo-Hlongwane1) [inline homework.md:59]: So I love how you were descriptive in your cloning, because on this one where are we copying it? The key idea here is directly being as descriptive as possibly like you had done in the cloning section

File: homework.md [hunk 5]
```diff
@@ -4,110 +4,110 @@
...
 
-It send it to a friend
+You download it to your computer
 
 ## Question 11
 
 What is a Pull Request?
 
-When you send a file over the internet
+Pull request means submitting one's work so that someone can look into it and make possible changes
```
Comment (Ntshembo-Hlongwane1) [inline homework.md:71]: Pull request is not entirely this, the idea within pull request is you alerting the owner/owners of a particular repo that you have made changes within their own repo so that they can review those changes and potentially merge your changes or discard those changes

**Comment by aashiqabrahams123** [comment]
Ntshembo Hlongwane, Pull requests let you tell others about changes you've pushed to a branch in a repository on GitHub.

**Comment by Ntshembo-Hlongwane1** [comment]
@aashiqabrahams123 Don't memorize google definition what you saying and what I'm saying are no different just phrased differently
